lib/anonymous.js

```diff
@@ -0,0 +1,214 @@
+import { useMutation, useQuery } from '@apollo/client'
+import { GraphQLError } from 'graphql'
+import { gql } from 'graphql-tag'
+import { useCallback, useEffect, useState } from 'react'
+import { useShowModal } from '../components/modal'
+import { Invoice as QrInvoice } from '../components/invoice'
+import { QrSkeleton } from '../components/qr'
+import { useMe } from '../components/me'
+import { msatsToSats } from './format'
+import FundError from '../components/fund-error'
+import { INVOICE } from '../fragments/wallet'
+import InvoiceStatus from '../components/invoice-status'
+import { sleep } from './time'
+import { Button } from 'react-bootstrap'
+import { CopyInput } from '../components/form'
+
+const Contacts = ({ invoiceHash }) => {
+  const subject = `Support request for payment hash: ${invoiceHash}`
+  const body = 'Hi, I successfully paid for <insert action> but the action did not work.'
+  return (
+    <div className='d-flex flex-column justify-content-center'>
+      <span>Payment hash</span>
+      <div className='w-100'>
+        <CopyInput type='text' placeholder={invoiceHash} readOnly noForm />
+      </div>
+      <div className='d-flex flex-row justify-content-center'>
+        <a
+          href={`mailto:kk@stacker.news?subject=${subject}&body=${body}`} className='nav-link p-0 d-inline-flex'
+          target='_blank' rel='noreferrer'
+        >
+          e-mail
+        </a>
+        <span className='mx-2 text-muted'> \ </span>
+        <a
+          href='https://tribes.sphinx.chat/t/stackerzchat' className='nav-link p-0 d-inline-flex'
+          target='_blank' rel='noreferrer'
+        >
+          sphinx
+        </a>
+        <span className='mx-2 text-muted'> \ </span>
+        <a
+          href='https://t.me/k00bideh' className='nav-link p-0 d-inline-flex'
+          target='_blank' rel='noreferrer'
+        >
+          telegram
+        </a>
+        <span className='mx-2 text-muted'> \ </span>
+        <a
+          href='https://simplex.chat/contact#/?v=1-2&smp=smp%3A%2F%2F6iIcWT_dF2zN_w5xzZEY7HI2Prbh3ldP07YTyDexPjE%3D%40smp10.simplex.im%2FebLYaEFGjsD3uK4fpE326c5QI1RZSxau%23%2F%3Fv%3D1-2%26dh%3DMCowBQYDK2VuAyEAV086Oj5yCsavWzIbRMCVuF6jq793Tt__rWvCec__viI%253D%26srv%3Drb2pbttocvnbrngnwziclp2f4ckjq65kebafws6g4hy22cdaiv5dwjqd.onion&data=%7B%22type%22%3A%22group%22%2C%22groupLinkId%22%3A%22cZwSGoQhyOUulzp7rwCdWQ%3D%3D%22%7D' className='nav-link p-0 d-inline-flex'
+          target='_blank' rel='noreferrer'
+        >
+          simplex
+        </a>
```

Comment:
The links for sphinx and simplex are group chats. Should we replace them with DM links like for TG?

Reply:
Let's not make more work for ourselves yet. This is good enough.